src/main/java/com/parasoft/findings/jenkins/coverage/model/SafeFraction.java

```diff
@@ -37,6 +38,7 @@
  */
 public class SafeFraction {
     private final Fraction fraction;
+    private final FilteredLog log = new FilteredLog();
```

Comment:
Using the FilteredLog instead of our custom Logger.getLogger() ensures consistency across the 'coverage' package.